src/js/trove/global.js

```diff
@@ -26,10 +26,22 @@
                 "flatness":0,
                 "name":"print",
                 "typ":["arrow", ["Any"], "Any"]},
-      "test-print": ["arrow", ["Any"], "Any"],
-      "print-error": ["arrow", ["Any"], "Any"],
-      "display": ["arrow", ["Any"], "Any"],
-      "display-error": ["arrow", ["Any"], "Any"],
+      "test-print": {"bind":"fun",
+                     "flatness":0,
+                     "name":"test-print",
+                     "typ":["arrow", ["Any"], "Any"]},
```

Comment:
These are verbose.  I wonder if we should add another abbreviation, `"flatarrow"`, that is specially handled by `type-util.js`.